decryptor/mysql/packet.go

```diff
@@ -15,7 +15,7 @@ const (
 	// SSL_REQUEST - https://dev.mysql.com/doc/internals/en/capability-flags.html#flag-CLIENT_SSL
 	SSL_REQUEST = 0x00000800
 	// https://dev.mysql.com/doc/internals/en/capability-flags.html#flag-CLIENT_DEPRECATE_EOF - 0x1000000
-	CLIENT_DEPRECATE_EOF = 1 << 6
+	CLIENT_DEPRECATE_EOF = 0x01000000
```

Comment:
why do you change it?

Reply:
because 1<<6 it's `0b1000000` (correct was 1<<8 for 2 byte comparison) and `0x01000000` it's `0b1000000000000000000000000` and I changed comparison. before, we compared flag with 2 byte integer value. now used full compatibility flag that has 4 byte size.